ext/groonga/rb-grn-table.c

```diff
@@ -184,7 +184,8 @@ rb_grn_table_inspect_content (VALUE self, VALUE inspected)
 }
 
 /*
- * テーブルの中身を人に見やすい文字列で返す。
+ * This function return contents of a table as a
+ * pretty-printed string.
```

Comment:
In Ruby, "pretty-print" is used for https://docs.ruby-lang.org/en/2.6.0/PP.html
It's different with "inspect": https://docs.ruby-lang.org/en/2.6.0/Object.html#method-i-inspect

Reply:
Thank you for your comment.
I've fixed it in c09caa53e7c34c768558fd3249ca355da19a3c1c.